src/config/laravel-omnipay.php

```diff
@@ -36,9 +36,16 @@
         'free' => [
             'driver'  => 'free_driver',
             'options' => [
-                'displayName'   => 'Kostenlos',
+                'displayName'   => 'Free',
```

Comment:
Müssten das nicht auch übersetzbare Texte sein?

Reply:
is not easy since __() seems to be unavailable when the config is loaded.
further investigation in https://github.com/Lan2Play/eventula-manager/issues/103